balancer/rls/picker.go

```diff
@@ -252,6 +252,26 @@ func (p *rlsPicker) handleRouteLookupResponse(cacheKey cacheKey, targets []strin
 	// entry would be used until expiration, and a new picker would be sent upon
 	// backoff expiry.
 	now := time.Now()
+
+	// "An RLS request is considered to have failed if it returns a non-OK
+	// status or the RLS response's targets list does not contain any non-empty
+	// entries." - RLS LB Policy design.
+	var nonEmptyTarget bool
+	for _, target := range targets {
+		if target != "" {
+			nonEmptyTarget = true
+			break
+		}
+	}
+	if !nonEmptyTarget {
```

Comment:
Let's try to avoid negatively named bools and especially negating negatives.  (tott/685)

E.g. name it `hasValidTarget` or reverse it:

```go
allTargetsEmpty := true
for _, target := range targets {
	if target != "" {
		allTargetsEmpty = false
		break
	}
}
if allTargetsEmpty {
```

Reply:
Hmmmm ok noted.